.codeclimate.yml

```diff
@@ -1,18 +1,15 @@
----
 exclude_paths:
 - ".git/"
 - "**.xml"
 - "**.yaml"
 - "**.yml"
-- "locale/"
-- "spec/"
-- "tools/"
+- locale/
+- spec/
+- tools/
 engines:
```

Comment:
Based on the code climate error:

> ```
> ERROR: only use one of 'engines', 'plugins'
> WARNING: missing 'version' key. Please add `version: "2"`
> WARNING: 'engines' has been deprecated, please use 'plugins' instead
> ```

These all need to be converted to plugins as well:

https://docs.codeclimate.com/docs/list-of-engines#official-plugins
https://docs.codeclimate.com/docs/list-of-engines#community-supported-plugins

Reply:
https://github.com/ManageIQ/manageiq-style/pull/13